tsdb/engine/tsm1/wal.go

```diff
@@ -383,20 +387,17 @@ func (l *WAL) writeToLog(entry WALEntry) (int, error) {
 	// limit how many concurrent encodings can be in flight.  Since we can only
```

Comment:
Yes.  I didn't see these comments earlier before I merged.  I'll submit a new PR to remove these lines.